proxy/start_container_interceptor.go

```diff
@@ -11,7 +11,24 @@ import (
 
 type startContainerInterceptor struct{ proxy *Proxy }
 
+func callWeaveAndLog(container *docker.Container, description string, args ...string) error {
+	if _, stderr, err := callWeave(args...); err != nil {
+		return fmt.Errorf("Failed: %s container %s: %s", description, container.ID, string(stderr))
+	} else if len(stderr) > 0 {
+		Log.Warningf("%s container %s: %s", description, container.ID, string(stderr))
+	}
+	return nil
+}
+
 func (i *startContainerInterceptor) InterceptRequest(r *http.Request) error {
+	if strings.HasSuffix(r.URL.Path, "restart") {
```

Comment:
we only get here after the regexp has established it is either 'start' or 'restart' at the very end of the string.